README.md

```diff
@@ -30,7 +30,7 @@ Abiwan will support multiple Cairo compiler versions, but not in parallel - diff
 | Abiwan                                                        | Cairo compiler                                                                                                                                               |
 | ------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------ |
 | [1.0.3](https://www.npmjs.com/package/abi-wan-kanabi/v/1.0.3) | [Cairo v1.0.0](https://github.com/starkware-libs/cairo/releases/tag/v1.0.0) <br> [Cairo v1.1.0](https://github.com/starkware-libs/cairo/releases/tag/v1.1.0) |
-| [2.1.1](https://www.npmjs.com/package/abi-wan-kanabi/v/2.1.1) | [Cairo v2.3.0](https://github.com/starkware-libs/cairo/releases/tag/v2.3.0)                                                                                  |
+| [2.2.0](https://www.npmjs.com/package/abi-wan-kanabi/v/2.2.0) | [Cairo v2.4.4](https://github.com/starkware-libs/cairo/releases/tag/v2.4.4)                                                                                  |
```

Comment:
is it better to maybe just add this new column for 2.4.4 support below 2.3.0, instead of replacing it?

Reply:
My logic is that there is since there is no breaking changes between the two versions, I just override but thinking about it, it's probably better to keep them both